webviz_subsurface/plugins/_well_cross_section_fmu.py

```diff
@@ -532,9 +532,13 @@ def _render_section(
             layout.update(self.plotly_theme["layout"])
             if "keep_zoom_state" in options:
                 layout["uirevision"] = "keep"
-            fencespec: List[List[float]] = [
-                [coord[0], coord[1]] for coord in xsect.fence
-            ]
+            # [[]] "solves" type hint, but is pretty dirty
+            # and probably wrong...
```

Comment:
what to do? 🤔

Reply:
Will not mypy allow Initializing as empty list ?
Tested quickly on 3.6 and 3.8 with no errors.
```python
fencespec: List[List[float]] = []
if xsect.fence is not None:
    fencespec = [[coord[0], coord[1]] for coord in xsect.fence]
```

Reply:
that is likely, probably didn't try 😉